arithmoi.cabal

```diff
@@ -105,9 +105,21 @@ test-suite spec
   type:                 exitcode-stdio-1.0
   hs-source-dirs:       test-suite
   ghc-options:          -Wall
-  main-is:              Spec.hs
+  main-is:              Test.hs
   default-language: Haskell2010
   build-depends:        base
-                        ,hspec
-                        ,arithmoi
+                        , arithmoi
+                        , tasty
+                        , tasty-smallcheck
+                        , tasty-quickcheck
+                        , tasty-hunit
+                        , QuickCheck
+                        , smallcheck
```

Comment:
what lower/upper bounds on these versions should we do? It'd be nice to make sure that we *know* that the deps they are ..
it'd be nice to have *some* mechanism for reconstituting when this is buildable.
Like, at the very least workingout what bounds on quickcheck/small check are "definitely" valid, and *lowerbounds* on deps. 

also why did you switch the test suite code to tasty rather than hspec? I dont really care, but would like some extra color